### PR TITLE
Create flag for media array changing

### PR DESCRIPTION
**Pull request by kristen**
#118

### CODE REVIEW

File: Mapstagram/AppDelegate.swift [hunk 1]
```diff
@@ -26,6 +26,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         UINavigationBar.appearance().tintColor = UIColor.whiteColor()
         UINavigationBar.appearance().barTintColor = mapstagramTintColor
         UINavigationBar.appearance().titleTextAttributes = [NSForegroundColorAttributeName: UIColor.whiteColor()]
+        UINavigationBar.appearance().translucent = false
```
Comment (kristen) [inline Mapstagram/AppDelegate.swift]: I snuck this one in...

File: Mapstagram/Services/SessionService.swift [hunk 2]
```diff
@@ -10,6 +10,18 @@ import Foundation
 import CoreLocation
 
 class SessionService {
+    enum MapstagramViews: Printable {
+        case Map, List, Grid
+        
+        var description: String {
```
Comment (kristen) [inline Mapstagram/Services/SessionService.swift]: We can remove this later, I just wanted to see which view was being reloaded in print statements

Comment (jamietsao) [inline Mapstagram/Services/SessionService.swift]: Sounds good.

Kinda lame that Swift enums don't just use their declarations as a default description.

**Comment by jamietsao** [comment]
Looks good - nice work.  Merge it!

**Pull request merged**